### PR TITLE
Add role for HomeAssistant

### PR DESCRIPTION
**Pull request by kenyonj**
This PR:
  - Adds the proper community.yml entry
  - Adds the role tasks for installing homeassistant
  - Adds an entry to the README
  - Adds appveyor entry

Why?
  - This is useful for allowing HomeAssistant to be installed with the
community scripts

### CODE REVIEW

**Comment by saltydk** [comment]
I will not be merging roles with localhost bindings going forward. If the image has the port exposed in its dockerfile it will work with nginx-proxy fine without it. If not, request it to be added on their repo.

File: roles/homeassistant/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      LETSENCRYPT_EMAIL: "{{ user.email }}"
+      CLEAN_TMP_DIR: "1"
+      ENABLE_CJK_FONT: "1"
+    volumes:
+      - "/etc/localtime:/etc/localtime:ro"
+      - "/opt/homeassistant:/config:rw"
+    devices: "{{ '/dev/dri:/dev/dri' if (gpu.intel and dev_dri.stat.exists) | default(false) else omit }}"
+    labels:
+      "com.github.cloudbox.cloudbox_managed": "true"
+    networks:
```
Comment (kenyonj) [inline roles/homeassistant/tasks/main.yml:55]:
```suggestion
    netowork_mode: host
    networks:
```

File: roles/homeassistant/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      - "/etc/localtime:/etc/localtime:ro"
+      - "/opt/homeassistant:/config:rw"
+    devices: "{{ '/dev/dri:/dev/dri' if (gpu.intel and dev_dri.stat.exists) | default(false) else omit }}"
+    labels:
+      "com.github.cloudbox.cloudbox_managed": "true"
+    network_mode: host
+    networks:
+      - name: cloudbox
+        aliases:
+          - homeassistant
```
Comment (kenyonj) [inline roles/homeassistant/tasks/main.yml:59]:
```suggestion
```

**Comment by kenyonj** [comment]
Thanks, I'll open this back up when I figure out how to get it all to work without that published port.

**Comment by saltydk** [comment]
Ask the developer of the image to add this to the dockerfile.
```
EXPOSE 8123
```
Example of one that uses expose: https://gitlab.com/sc4h/docker-qbittorrent/-/blob/master/Dockerfile

**Comment by kenyonj** [comment]
The thing is, homeassistant really needs `--net=host` or for ansible/composer it should be `network_mode: host` and that doesn't really jive with the spirit of cloudbox and doesn't really work with nginx reverse proxying. I think I might just have to manually manage this with portainer. Thanks for your help though!

**Pull request closed without merging**